### ISSUE TITLE
Headless: `player.country` property

### ISSUE THREAD

**Issue by olegmisar** [haxball/haxball-issues#351]
I am currently working on automatic host. Players pick team just using words and my bot can understand them. But if someone decides to pick player by nationality, my bot will think it's nickname. Please add `player.country` property.

**Comment by olegmisar** [comment]
@basro I don't think it is hard to implement this feature but I've (and not only I) waited for a while. Please add it.

**Comment by thenorthstar** [comment]
For those who are newly seeing this issue, it's possible through editing the HaxBall Headless API.

Output: https://thehax.pl/forum/powtorki.php?nagranie=8c65164999f20d47faaec5da59b7c911

**Comment by byoxygen** [comment]
> For those who are newly seeing this issue, it's possible through editing the HaxBall Headless API.
> 
> Output: https://thehax.pl/forum/powtorki.php?nagranie=8c65164999f20d47faaec5da59b7c911

@thenorthstar I don't know how to edit API.
